### PR TITLE
[INLONG-10365][Audit] Optimizing TCP sticky packets may lead to duplication of audit data 

### LINKED ISSUE #10365: [Improve][Audit] Optimizing TCP sticky packets may lead to duplication of audit data
### Description

Optimizing TCP sticky packets may lead to duplication of audit data.

### InLong Component

InLong Audit

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by doleyzi**
- Fixes #10365

### Motivation

Optimizing TCP sticky packets may lead to duplication of audit data 

### Modifications

Use the native TCP socket instead of the Netty to reduce program load.

### CODE REVIEW

**Review by luchunliang — APPROVED** [review]

File: inlong-audit/audit-sdk/src/main/java/org/apache/inlong/audit/send/SenderManager.java [hunk 1]
```diff
@@ -20,190 +20,201 @@
...
+
+            byte[] header = new byte[PACKAGE_HEADER_LEN];
+            readFully(inputStream, header, PACKAGE_HEADER_LEN);
+
+            int bodyLen = ((header[0] & 0xFF) << 24) |
+                    ((header[1] & 0xFF) << 16) |
+                    ((header[2] & 0xFF) << 8) |
+                    (header[3] & 0xFF);
+
+            byte[] body = new byte[bodyLen];
```
Comment (justinwwhuang) [inline inlong-audit/audit-sdk/src/main/java/org/apache/inlong/audit/send/SenderManager.java:147]: Do we need to determine if bodyLen is within a reasonable range here

Comment (doleyzi) [reply inlong-audit/audit-sdk/src/main/java/org/apache/inlong/audit/send/SenderManager.java:153]: Fixed

File: inlong-audit/audit-sdk/src/main/java/org/apache/inlong/audit/util/RequestIdUtils.java [hunk 2]
```diff
@@ -17,33 +17,22 @@
...
-     * Constructor
-     *
-     * @param sendIp
-     * @param sendPort
-     * @param result
-     */
-    public SenderResult(String sendIp, int sendPort, boolean result) {
-        this.addr = new InetSocketAddress(sendIp, sendPort);
-        this.result = result;
+    public static Long nextRequestId() {
```
Comment (aloyszhang) [inline inlong-audit/audit-sdk/src/main/java/org/apache/inlong/audit/util/RequestIdUtils.java:30]: Race conditions may happen here.

File: inlong-audit/audit-sdk/src/main/java/org/apache/inlong/audit/AuditReporterImpl.java [hunk 3]
```diff
@@ -385,16 +399,14 @@ private void summaryExpiredStatMap(long isolateKey) {
      * Summary pre stat map
      */
     private void summaryPreStatMap(long isolateKey, ConcurrentHashMap<String, StatInfo> statInfo) {
-        List<String> expiredKeys = this.expiredKeyList.computeIfAbsent(isolateKey, k -> new ArrayList<>());
+        HashSet<String> expiredKeys = this.expiredKeyList.computeIfAbsent(isolateKey, k -> new HashSet<>());
```
[the remark this answers is not in the document]
Comment (doleyzi) [reply inlong-audit/audit-sdk/src/main/java/org/apache/inlong/audit/AuditReporterImpl.java]: Fixed

File: inlong-audit/audit-sdk/src/main/java/org/apache/inlong/audit/util/RequestIdUtils.java (hunk 2 above)
Comment (doleyzi) [reply inlong-audit/audit-sdk/src/main/java/org/apache/inlong/audit/util/RequestIdUtils.java:30]: It doesn't matter, requestId is just used as a request ID. When the sending fails, the data that marks the SDK instance fails. In addition, only when it is 0, does global duplication occur, but each SDK instance is obtained by a single thread, and duplication is impossible.
1. Both agent and dataproxy are single instances and only operate with one thread.
2. Sort uses multiple instances. Each instance is single-threaded. A single thread obtains the RequestId serially without duplication.

**Review by aloyszhang — APPROVED** [review]

**Review by justinwwhuang — APPROVED** [review]
LGTM

**Pull request merged**